src/middlewares/auth.js

```diff
@@ -1,59 +1,61 @@
-/* eslint-disable prefer-destructuring */
+import { config } from 'dotenv';
 import jwt from 'jsonwebtoken';
 
-const auth = {
-  generateToken(user) {
+config();
+
+export default {
+
+  generateToken(newUser) {
+    const {
+      id, first_name, last_name, email, role
```

Comment:
I don't think there is any need to sign the first and last names. Moving forward, we do not use it. But that's not worth slitting hairs over though.

Reply:
It is always good to sign your token with as much details as needed. Of course, without using the password, LOL. This way, just decoding your token alone will allow me see everything about you without making a request to the DB. For example, signing only the `email address and user id`, will require me to call the DB to see your `permission levels`. One wasted DB request.

Reply:
For the `role`, I asked Oluchi to sign it. 
But like I said, it's not worth splitting hairs over.